.github/config.yml

```
@@ -4,22 +4,36 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: |
-  Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
-  If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+  Thanks for opening your first issue! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.  
+  Please follow the issue template to help us help you 👍🎉😄   
+  If you have screenshots [or a gif](https://publiclab.org/gifs/) to share demonstrating the issue, that's really helpful! 📸
+  Do join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
- Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ Also please refer [here](https://github.com/publiclab/PublicLab.Editor#installation) for installation help 💿    
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄    
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to https://publiclab.org in the next few days. 
- In the meantime, can you tell us your Twitter handle so we can thank you properly?
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well! In the meantime, can you tell us your Twitter handle so we can thank you properly?
```

Ah this won't happen quite as fast as on `plots2` -- it'll trigger Dependabot to open a new version in a PR on `plots2` /if/ we publish it to NPM, and then it'll need to be merged there. So, maybe just end with "in a few days"?

Like here: 

https://github.com/publiclab/leaflet-blurred-location/blob/1bff37cb96518bd87192277b14c290cba3c138ba/.github/config.yml#L20

```suggestion
 Your code will likely be published to PublicLab.org in the next few days. In the meantime, can you tell us your Twitter handle so we can thank you properly?
```